contracts/libraries/CircleCCTPAdapter.sol

```diff
@@ -81,7 +81,20 @@ abstract contract CircleCCTPAdapter {
     function _transferUsdc(address to, uint256 amount) internal {
         // Only approve the exact amount to be transferred
         usdcToken.safeIncreaseAllowance(address(cctpTokenMessenger), amount);
-        // Submit the amount to be transferred to bridged via the TokenMessenger
-        cctpTokenMessenger.depositForBurn(amount, recipientCircleDomainId, _addressToBytes32(to), address(usdcToken));
+        // Submit the amount to be transferred to bridged via the TokenMessenger.
+        // If the amount to send exceeds the burn limit per message, then split the message into smaller parts.
+        ITokenMinter cctpMinter = cctpTokenMessenger.localMinter();
```

Comment:
We may want to require that the CCTPtoken bridge is enabled otherwise we're going to potentially try to resolve .localMinter() from address(0)

Reply:
I don't think that's necessary as `depositForBurn` will revert if this is 0x0 [here](https://github.com/circlefin/evm-cctp-contracts/blob/817397db0a12963accc08ff86065491577bbc0e5/src/TokenMessenger.sol#L441)